### PR TITLE
Mapped lots of random things

### PR DESCRIPTION
**Pull request by Shnupbups**
Updated the original PR for 1.14.4 proper

### CODE REVIEW

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/client/input/KeyCombo.mapping [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_4324 net/minecraft/client/input/KeyCombo
```
Comment (Runemoro) [inline mappings/net/minecraft/client/input/KeyCombo.mapping]: This class was already mapped by my Realms PR.

File: mappings/net/minecraft/client/network/packet/BlockPlayerActionS2CPacket.mapping [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4463 net/minecraft/client/network/packet/BlockPlayerActionS2CPacket
+	FIELD field_20319 action Lnet/minecraft/class_2846$class_2847;
+	FIELD field_20320 logger Lorg/apache/logging/log4j/Logger;
```
Comment (Runemoro) [inline mappings/net/minecraft/client/network/packet/BlockPlayerActionS2CPacket.mapping]: Isn't this a `static final` field? If yes, it should be `LOGGER`.

File: mappings/net/minecraft/client/input/KeyCombo.mapping (hunk 1 above)
Comment (Shnupbups) [reply mappings/net/minecraft/client/input/KeyCombo.mapping]: Oops, missed that one! Removed.

File: mappings/net/minecraft/client/network/packet/BlockPlayerActionS2CPacket.mapping (hunk 2 above)
Comment (Shnupbups) [reply mappings/net/minecraft/client/network/packet/BlockPlayerActionS2CPacket.mapping]: Right, whoops. Fixed.

**Review by modmuss50 — APPROVED** [review]
Looks good, sorry for taking a while to review it.

**Pull request merged**